### PR TITLE
Update already-HOC-registered text on signup page

### PR DESCRIPTION
**Pull request by ewjordan**
Now that HOC is over, this PR changes text on /users/sign_up for people who are already registered for HOC 2016 but don't have an account from:

screen shot 2016-12-09 at 3 39 52 pm

to

screen shot 2016-12-09 at 3 17 29 pm

### CODE REVIEW

**Comment by ewjordan** [comment]
Not going to merge to staging until at least Monday (this is not a critical update as HOC finishes)

**Comment by ewjordan** [comment]
Also, the layout difference is not real, I accidentally had my zoom level set when screenshotting, the zero-zoom one looks like this:

screen shot 2016-12-09 at 4 01 49 pm

File: dashboard/config/locales/en.yml [hunk 1]
```diff
@@ -104,8 +104,8 @@ en:
...
-    hoc_already_signed_up_heading: 'Get started without logging in'
-    hoc_already_signed_up_content: 'You are signed up to teach an Hour of Code! You and your students do not need to sign in to Code Studio to participate in this event. To get started, <a href="https://code.org/learn">choose a tutorial</a> for your classroom and share the link with them.<br/><br/>If you enjoyed the Hour of Code and want to go further, you can set up a password and create a Code Studio account below to enroll your students in any of our <a href="https://code.org/educate">longer courses</a>.'
+    hoc_already_signed_up_heading: 'Sign up for Code.org'
+    hoc_already_signed_up_content: 'Thank you for hosting an Hour of Code. To sign in to Code Studio, you’ll need to set up a password for  your account. This will let you save your progress and manage your classroom. You’ll be able to assign courses to students and track their progress. <a href="https://studio.code.org/">You can also continue to browse the various stages and puzzles</a> without creating an account.'
```
Comment (tanyaparker) [inline dashboard/config/locales/en.yml]: Extra space between for and your in `set up a password for  your account`

**Comment by tanyaparker** [comment]
If we plan on using the HOC strings again next year, I would prefer to keep both strings so that we can keep both translations. If we keep switching back and forth we'll lose translations. We could create two keys `pre_hoc_already_signed_up_content` and `post_hoc_already_signed_up_content` and switch those keys.

Also, we want to remove URLs from source strings as much as possible. Yes, people could still put in other HTML in their translation but this is one of our first steps to try and remove the possibility for a translator to add a bad URL. After Hour of Code we'll take next steps to do more HTML sanitization. 

It would look something like this:
- `hoc_already_signed_up_content: "...<a href="%{studio_url}">You can also continue to browse the various stages and puzzles</a> without creating..."`

- `%p= t('signup_form.hoc_already_signed_up_content', studio_url: CDO.studio_url('/')).html_safe`

**Comment by tanyaparker** [comment]
@caleybrock this is the other change that should go out today. For path of least resistance as long as we fix the double space typo this is *fine* to go out, and we can do the other i18n fixes after.

**Pull request merged**